pkg/build/builder/util_linux.go

```diff
@@ -98,3 +104,24 @@ func getCgroupParent() (string, error) {
 	glog.V(6).Infof("found cgroup values map: %v", cgMap)
 	return extractParentFromCgroupMap(cgMap)
 }
+
+// getSELinuxLabel retrieves the selinux label associated w/ this container.
+func getSELinuxLabel() (string, error) {
+	f, err := os.Open("/proc/mounts")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+	s := bufio.NewScanner(f)
+
+	for s.Scan() {
+		if err := s.Err(); err != nil {
+			return "", err
+		}
+		if match := selinuxLabelPattern.FindStringSubmatch(s.Text()); match != nil {
+			glog.V(6).Infof("found selinux labels: %v", match[1])
+			return fmt.Sprintf("label=level:%s", match[1]), nil
```

Comment:
I thought we needed to match user/role/type as well here?  What happens if there is a customer SELinux label on the container, no match?

Reply:
s/customer/custom but in any case, response was explained here: https://github.com/openshift/origin/pull/17094#issuecomment-340793534